backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/utils/CpuPinningHelper.java

```diff
@@ -92,6 +94,25 @@
         }
     }
 
+    /**
+     * Provides a CPU pinning string based on the VdsCpuUnit list.
+     * The pinning will be based on the list order.
+     *
+     * @param vdsCpuUnits a list of VdsCpuUnit
+     * @return a string of CPU pinning
+     */
+    public static String createCpuPinning(List<VdsCpuUnit> vdsCpuUnits) {
+        int vcpuNumber = 0;
```

Comment:
I guess this part could be removed.

Reply:
Yes, I left it for now to see it works when testing and then to remove it.